libs/langgraph/langgraph/errors.py

```diff
@@ -22,13 +22,22 @@ class GraphRecursionError(RecursionError):
         )
     """
 
-    pass
+    def __init__(
+        self,
+        message: str = "Recursion limit reached.\n\nTroubleshooting URL: https://python.langchain.com/docs/troubleshooting/errors/GRAPH_RECURSION_LIMIT/",
+    ):
+        self.message = message
+        super().__init__(self.message)
 
 
 class InvalidUpdateError(Exception):
     """Raised when attempting to update a channel with an invalid set of updates."""
 
-    pass
+    def __init__(self, message: str, code: Optional[str] = None):
+        if code is not None:
+            message = f"{message}\n\nTroubleshooting URL: https://python.langchain.com/docs/troubleshooting/errors/{code}/"
```

Comment:
also we need to fix the URLs here to point to langgraph docs

Reply:
No we're adding a redirect so we don't have github.io URLs in code